sources/3.0.0/FC/S-102_Ed_3.0.0_FC.xml

```diff
@@ -0,0 +1,122 @@
+						<S100CI:country>Principauté de Monaco</S100CI:country>
+						<S100CI:electronicMailAddress>info@iho.int</S100CI:electronicMailAddress>
+					</S100CI:address>
+					<S100CI:onlineResource>
+						<S100CI:linkage>www.iho.int</S100CI:linkage>
+					</S100CI:onlineResource>
+		</S100CI:contactInfo>
+			</S100CI:CI_Organisation>
+		</S100CI:party>
+	</S100FC:producer>
+	<S100FC:classification>unclassified</S100FC:classification>
+	<S100FC:S100_FC_DefinitionSources>
+		<S100FC:FC_DefinitionSource id="IHOREG">
+			<S100FC:source>
+				<S100CI:title>IHO GI Registry</S100CI:title>
+				<S100CI:onlineResource>
+					<S100CI:linkage>http://registry.iho.int</S100CI:linkage>
+				</S100CI:onlineResource>
+			</S100FC:source>
+		</S100FC:FC_DefinitionSource>
+	</S100FC:S100_FC_DefinitionSources>
+	<S100FC:S100_FC_SimpleAttributes>
+		<S100FC:S100_FC_SimpleAttribute>
+			<S100FC:name>Depth</S100FC:name>
+			<S100FC:definition>The vertical distance from a given water level to the bottom.</S100FC:definition>
+			<S100FC:code>depth</S100FC:code>
+			<S100FC:alias>DEPTH</S100FC:alias>
+			<S100FC:definitionReference>
+				<S100FC:sourceIdentifier>820</S100FC:sourceIdentifier>
+				<S100FC:definitionSource ref="IHOREG"/>
+			</S100FC:definitionReference>
+			<S100FC:valueType>real</S100FC:valueType>
+		</S100FC:S100_FC_SimpleAttribute>
+		<S100FC:S100_FC_SimpleAttribute>
+			<S100FC:name>Uncertainty</S100FC:name>
+			<S100FC:definition>The interval (about a given value) that will contain the true value of the measurement at a specific confidence level.</S100FC:definition>
+			<S100FC:code>uncertainty</S100FC:code>
+			<S100FC:remarks>Represents a +/- value defining the possible range of associated depth, expressed as a positive number.</S100FC:remarks>
+			<S100FC:definitionReference>
+				<S100FC:sourceIdentifier>884</S100FC:sourceIdentifier>
+				<S100FC:definitionSource ref="IHOREG"/>
+			</S100FC:definitionReference>
+			<S100FC:valueType>real</S100FC:valueType>
+		</S100FC:S100_FC_SimpleAttribute>
+	</S100FC:S100_FC_SimpleAttributes>
+	<S100FC:S100_FC_FeatureTypes>
+		<S100FC:S100_FC_FeatureType isAbstract="false">
+			<S100FC:name>Bathymetry Coverage</S100FC:name>
+			<S100FC:definition>A set of value items required to define a dataset representing an depth calculation and its associated uncertainty.</S100FC:definition>
+			<S100FC:code>BathymetryCoverage</S100FC:code>
+			<S100FC:definitionReference>
+				<S100FC:sourceIdentifier>401</S100FC:sourceIdentifier>
+				<S100FC:definitionSource ref="IHOREG"/>
+			</S100FC:definitionReference>
+			<S100FC:attributeBinding sequential="false">
+				<S100FC:multiplicity>
+					<S100Base:lower>1</S100Base:lower>
+					<S100Base:upper xsi:nil="false" infinite="false">1</S100Base:upper>
+				</S100FC:multiplicity>
+				<S100FC:attribute ref="depth"/>
+			</S100FC:attributeBinding>
+			<S100FC:attributeBinding sequential="false">
+				<S100FC:multiplicity>
+					<S100Base:lower>1</S100Base:lower>
```

Comment:
The `uncertainty` is optional since S-102 ed. 2.2.0. So the lower bound must be `0` not `1`.
See section 11.2.7.  -> BathymetryCoverage feature instance group — values dataset

Reply:
Good catch--thank you!